domains/yeasin.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "yeasin.is-a.dev",
+    "owner": {
+        "username": "yeasiin",
+        "email": "iamyeasiin@gmail.com"
+    },
+    "record": {
+        "CNAME": "yeasiin.github.io/website"
```

`CNAME` record must contain a domain name only.
```suggestion
        "CNAME": "yeasiin.github.io"
```